workshop/content/docs/api-deep-dive/tiles.md

```diff
@@ -566,6 +566,78 @@ Map tile:
 
 You can see [here](https://maps.gnosis.earth/ogcapi/collections/blueMarble/map/tiles?f=json) an example of a request for a (map) tileset list and [here](https://maps.gnosis.earth/ogcapi/collections/blueMarble/map/tiles/GoogleCRS84Quad?f=json) an example of a request for (map) tileset metadata.
 
+### Client usage
+
+Various software supports the OGC API - Tiles standard (see <https://github.com/opengeospatial/ogcapi-tiles/blob/master/implementations.adoc#clients> for a list of supporting implementations).
```

Comment:
@krishnaglodha We already have a link for implementations on [this](https://github.com/opengeospatial/ogcapi-workshop/commit/1b45e2e36dcbdbaf77643d49addd953a2d4f93f5#diff-191239ca8f7e559fb10e7c82d6082a76c39c4f072737a25171ed324b6fe68366R68) section :-) No need to repeat it, here!